### ISSUE TITLE
Mac OS Catalina klogg did not open files from finder

### ISSUE THREAD

**Issue by raizon** [variar/klogg#123]
when I double click on file klogg is starting, but empty.
I can open file only by drag n drop or via dialog.

**Comment by variar** [comment]
Unfortunately, I don't have access to Mac OS. Fixing this will require some research.

**Comment by raizon** [comment]
I can give remote access to my machine if you need it.

**Comment by variar** [comment]
Thanks, but I'm travelling at the moment. Mac OS integration has been copied from glogg almost as-is. I'll try to figure out how it is supposed to work. The issue might be connected with this code in plist
```
<key>CFBundleDocumentTypes</key>
    <array>
            <dict>
                    <key>CFBundleTypeName</key>
                    <string>Plain Text File</string>
                    <key>CFBundleTypeRole</key>
                    <string>Viewer</string>
                    <key>LSItemContentTypes</key>
                    <array>
                            <string>public.text</string>
                    </array>
            </dict>
    </array>
```

**Comment by variar** [comment]
@raizon does it work from "Open with" menu?

**Comment by variar** [comment]
I was able to reproduce the issue on mac. Should be fixed soon

**Comment by variar** [comment]
Might be fixed in 19.11.0.519
